src/Plugin/search_api/backend/SearchApiElasticsearchBackend.php

```diff
@@ -189,6 +194,7 @@ public function defaultConfiguration() {
       'http_method' => 'AUTO',
       'autocorrect_spell' => TRUE,
       'autocorrect_suggest_words' => TRUE,
+      'fuzziness' => FALSE,
```

Comment:
Does it make sense to default fuzziness to enabled?

Reply:
Now that you mention it, yeah, it kinda does. I think I was being conservative, going for an opt in style feature update, but this module doesn't have a release yet on the 8.x-5.x branch, so lets do it.